tests/parametric_study/test_parametric_runner.py

```diff
@@ -431,3 +431,194 @@ def test_simulate_returns_empty_list_when_no_simulations_meet_criteria(
     for record in caplog.records:
         assert record.levelname == "WARNING"
         assert "None of the input simulations meet the criteria selected" in record.message
+
+
+@pytest.mark.parametrize(
+    "simulation_ids_input",
+    [
+        ["test_2"],
+        ["test_0", "test_2"],
+    ],
+)
+def test_simulate_filters_by_simulation_ids_if_the_list_has_atleast_one_valid_element(
```

Comment:
Maybe add a few more simulations and set the status to Completed in the internal dataframe of parametricstudy. Verify that only Pending simulations are passed to `mock_additive.simulate()`.